### ISSUE TITLE
Consider moving JUMPer documentation to JUMP Hub

### ISSUE THREAD

**Issue by shntnu** [broadinstitute/jump_hub#64]
Context: Anne suggests that the contents of the README of https://github.com/broadinstitute/monorepo/tree/main/libs/jump_rr should live in JUMP Hub

Anne Carpenter
12:04 PM Nov 1
This is a big suggestion, but I'd love the information at JUMPer tools to live on the same pages we are editing here. In other words, we have a how to for jupyter notebooks as a subpage here so I'd like a how to for the JUMP-rr tools here also. And the existing JUMP-rr page would point back here

Alan Fernando Muñoz González
9:36 AM Nov 15
I can duplicate it (at the risk of having one copy change over time), but I think it is crucial to have the info over there in our monorepo (as it is how people can access the programmatic resources). We can add some screenshots with usage examples here though.

Anne Carpenter
10:09 AM Nov 15
In my vast experience being practically elderly now, duplicating is always
a bad idea because as you say, they become out of sync no matter how well
intentioned we are.

Why not have a link over there at monorepo that points to this set of
pages, though? They key thing is that at each natural landing site or
location where people might look we either have the information OR a
pointer to the information. But we don't necessarily need a primary copy of
the information at each location.

Alan Fernando Muñoz González
10:24 AM Nov 15
Agree with avoiding duplication. That said, as a software developer, the code and documentation should be as close as possible. Having the documentation away from the code that produces JUMPer interfaces makes it hard to understand how to get into it to improve it/understand it from a technical perspective. One potential solution is to move the 'users manual' here with all the other resources, and keep the development side of things there. I would still feel a bit uneasy if user and developer documentation are not in the same place.

Alan Fernando Muñoz González
10:24 AM Nov 15
Worth a discussion with the rest of the group though!

**Comment by afermg** [comment]
This one relates to a broader discussion about whether our tools/datasets should be self-contained or they should depend on the Hub. Similarly to #67, it is not very actionable.

**Comment by shntnu** [comment]
Makes sense
Feel free to move it anywhere else that is more appropriate

**Comment by AnneCarpenter** [comment]
I strongly would like all documentation for a given tool to live in one place. 

Alán has a strong preference that it be near the code; I can live with that. 

So let's move the Usage section from here: 
https://broadinstitute.github.io/jump_hub/howto/0_howto_interactive_tools.html
(leaving behind a stub that says "For JUMPrr tools, see Usage instructions here[URL]".)

And add that Usage material as a subsection of this page:
https://github.com/broadinstitute/monorepo/tree/main/libs/jump_rr which would then be the URL to go into the above sentence.

WDYT?

I like that it avoids what I imagine would be two common scenarios: someone either goes straight to the JUMPrr tools page, or they go to this page of JUMP Hub and click JUMPrr tools and immediately leave the Hub, not realizing there's instructions there. 

![Image](https://github.com/user-attachments/assets/95e33beb-b2ea-434f-92e3-20ff244cfd88)

It also avoids the current problem that if someone finds themselves here:
https://github.com/broadinstitute/monorepo/tree/main/libs/jump_rr

Finally, we should edit a sentence here https://github.com/broadinstitute/monorepo/tree/main/libs/jump_rr 
from this: "You can read more about JUMP at the JUMP-Cell Painting Hub."
to this: "You can read more about JUMP data and find other useful tools at the JUMP-Cell Painting Hub."

**Comment by afermg** [comment]
I agree that having some JUMP_rr documentation in the hub may be confusing. I have left just enough documentation on the JUMP_rr  subsection of the monorepo and the tables themselves to make them usable. That said, I am not sure that we want to move other detailed tutorials out of the Hub, since that is the purpose of the hub is to concentrate (and make easily searchable) all the JUMP-related info. I still think that the step-by-step tutorials should live in the hub and we should have bidirectional links between them. 

I suggest this approach:
- Keep developer-facing docs on the jump_rr subsection of the monorepo.
- Add a new page on the Hub specifically for step-by-step tutorials of the jump_rr tables.
- Add a link to the JUMP_rr tutorials AND to the repository(quick links) in the landing page 

That does mean that the link themselves would be on the repository instead, but that + a very minimal instructions set should be enough to keep the repository compact and tidy. I will ask Paula about her take on this, since she has a fresher outlook.

i performed the update of your last point to make the sentence clearer.

**Comment by AnneCarpenter** [comment]
Ah, the fact that tutorials will soon exist is an interesting twist. side note: you can take the text for a compound search from what i just sent to our collaborator who asked, to use as a draft for that tutorial! 

I don't 100% love this solution because I worry the bidirectional links will be too easily overlooked. But, the tutorials being much lengthier than just the Usage section changes the game as to what can fit on the single jumprr subsection README of the monorepo! 

So I think your approach will work so long as we really strive to make the bidirectional links obvious (without making them blink or highlighted in neon color or comic sans :D )

**Comment by afermg** [comment]
Sounds like a plan. You didn't say anything about word art though 

![Image](https://github.com/user-attachments/assets/cf46b813-6f25-4217-80c4-eb5560177b93)

**Comment by afermg** [comment]
This has been covered by the [tutorial](https://broadinstitute.github.io/jump_hub/howto/1_jumprr_steps.html) written by @PaulaLlanos, we have comprehensive documentation, though she considered making a "quick reference" for JUMP_rr tables too.
